test/onnx/dynamo/test_exporter_api.py

```diff
@@ -0,0 +1,150 @@
+# Owner(s): ["module: onnx"]
+import io
+import unittest
+
+import onnx
+
+import torch
+from beartype import roar
+from torch.onnx import ExportOptions, ExportOutput, ExportOutputSerializer
+from torch.onnx._internal.exporter import (
+    _DEFAULT_OPSET_VERSION,
+    _resolve_export_options,
+)
+
+
+class SampleModel(torch.nn.Module):
+    def forward(self, x):
+        y = x + 1
+        z = y.relu()
+        return (y, z)
+
+
+class TestExportOptionsAPI(unittest.TestCase):
+    def test_opset_version_default(self):
+        options = _resolve_export_options(None)
+        self.assertEquals(options.opset_version, _DEFAULT_OPSET_VERSION)
+
+    def test_opset_version_explicit(self):
+        options = _resolve_export_options(ExportOptions(opset_version=3000))
+        self.assertEquals(options.opset_version, 3000)
+
+    def test_raise_on_invalid_argument_type(self):
+        expected_exception_type = roar.BeartypeException
+        with self.assertRaises(expected_exception_type):
+            options = ExportOptions(opset_version="3000")  # type: ignore[arg-type]
+        with self.assertRaises(expected_exception_type):
+            options = ExportOptions(dynamic_shapes=2)  # type: ignore[arg-type]
+        with self.assertRaises(expected_exception_type):
+            options = ExportOptions(logger="DEBUG")  # type: ignore[arg-type]
+        with self.assertRaises(expected_exception_type):
+            _resolve_export_options(options=12)  # type: ignore[arg-type]
+
+    def test_dynamic_shapes_default(self):
+        options = _resolve_export_options(None)
+        self.assertIsNone(options.dynamic_shapes)
+
+    def test_dynamic_shapes_explicit(self):
+        options = _resolve_export_options(ExportOptions(dynamic_shapes=None))
+        self.assertIsNone(options.dynamic_shapes)
+        options = _resolve_export_options(ExportOptions(dynamic_shapes=True))
+        self.assertTrue(options.dynamic_shapes)
+        options = _resolve_export_options(ExportOptions(dynamic_shapes=False))
+        self.assertFalse(options.dynamic_shapes)
+
+    def test_logger_default(self):
+        import logging
+
+        options = _resolve_export_options(None)
+        self.assertEquals(options.logger, logging.getLogger().getChild("torch.onnx"))
+
+    def test_logger_explicit(self):
+        import logging
+
+        options = _resolve_export_options(ExportOptions(logger=logging.getLogger()))
+        self.assertEquals(options.logger, logging.getLogger())
+        self.assertNotEquals(options.logger, logging.getLogger().getChild("torch.onnx"))
+
+
+class TestDynamoExportAPI(unittest.TestCase):
+    def test_default_export(self):
+        torch.onnx.dynamo_export(SampleModel(), torch.randn(1, 1, 2))
```

Comment:
```suggestion
        out = torch.onnx.dynamo_export(SampleModel(), torch.randn(1, 1, 2))
        self.assertTru(isinstance(out, ExportOutput)
```
usually we assert something on a test.